templates/includes/contact-us.html

```diff
@@ -10,141 +10,225 @@ <h2 class="p-modal__title" id="modal-title">Get your Kubeflow stack running</h2>
+          </label>
           </div>
           <div class="col-4 u-sv3">
-            <input type="checkbox" id="pain-points-Planning">
-            <label for="pain-points-Planning">Planning</label>
-            <input type="checkbox" id="pain-points-Performance">
-            <label for="pain-points-Performance">Performance</label>
-            <input type="checkbox" id="pain-points-Observability">
-            <label for="pain-points-Observability">Observability</label>
-            <input type="checkbox" id="pain-points-Security">
-            <label for="pain-points-Security">Security</label>
-            <input type="checkbox" id="pain-points-Compliance">
-            <label for="pain-points-Compliance">Compliance</label>
+            <label class="p-checkbox" for="pain-points-Planning">
+              <input type="checkbox" class="p-checkbox__input" id="pain-points-Planning">
+              <span class="p-checkbox__label">Planning</span>
+            </label>
+            <label class="p-checkbox" for="pain-points-Performance">
+              <input type="checkbox" class="p-checkbox__input" id="pain-points-Performance">
+              <span class="p-checkbox__label">Performance</span>
+            </label>
+            <label class="p-checkbox" for="pain-points-Observability">
+              <input type="checkbox" class="p-checkbox__input" id="pain-points-Observability">
+              <span class="p-checkbox__label">Observability</span>
+            </label>
+            <label class="p-checkbox" for="pain-points-Security">
+              <input type="checkbox" class="p-checkbox__input" id="pain-points-Security">
+              <span class="p-checkbox__label">Security</span>
+            </label>
+            <label class="p-checkbox" for="pain-points-Compliance">
+              <input type="checkbox" class="p-checkbox__input" id="pain-points-Compliance">
+              <span class="p-checkbox__label">Compliance</span>
+            </label>
             <div class="js-other-container">
-              <input type="checkbox" class="js-other-container__checkbox" id="pain-points-other">
-              <label for="pain-points-other">Other</label>
-              <input type="text" id="pain-points-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other private cloud platform">
+              <label class="p-checkbox" for="pain-points-other">
+                <input type="checkbox" class="js-other-container__checkbox p-checkbox__input" id="pain-points-other">
+                <span class="p-checkbox__label">Other</span>
+              </label>
+              <label for="pain-points-other-specified">
+                <input type="text" id="pain-points-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other private cloud platform">
+              </label>
             </div>
```

Comment:
The `Other`'s margin looks different from other checkboxes
![image](https://user-images.githubusercontent.com/57550290/152148271-eae5892c-61c1-468e-8664-275e76061eb3.png)

Reply:
Fixed